### ISSUE TITLE
Document a way to visualize OD data distribution 

### ISSUE THREAD

**Issue by shantanuchhabra** [apple/turicreate#3002]
The Turi Create Userguide for Object Detection doesn't provide a way for users to be able to visualize the data distribution (either by bounding box or by image). While it's easy to do for Image Classification (`data['label'].show()`), it's not that easy to do for Object Detection and it may be worth documenting.

Here is the code we should insert in our documentation somewhere at some point:

```python
def show_object_detection_bounding_box_distribution_by_label(sf):
     labels = sf['annotations'].apply(lambda A: [x['label'] for x in A])
     stacked = labels.stack()
     stacked['X1'].show()
```

**Comment by TobyRoseman** [comment]
I agree this would be useful to document. I literally wanted to do this exact same thing yesterday.

Here is how I got the labels from annotations. It might be a bit simpler.
`labels = sf['annotations'].stack().unpack()['label']`

Also rather than calling `labels.show()` I was using `labels.value_counts()`.
